ci/run_style_check

```diff
@@ -10,8 +10,10 @@ files=(
   spelling.txt
 )
 
+KVER="v$(grep --only-matching --perl-regexp 'KVER=\K(\d\.\d*)' .travis.yml | tail -n 1)"
+
```

Comment:
Fine with me.

The reason I used grep/sed is bc most people doesn't know a thing about perl regexp :))